manifests/repo/zypper.pp

```diff
@@ -0,0 +1,20 @@
+# PRIVATE CLASS: do not use directly
```

Comment:
please document the class with puppet-strings:
* Add a `@summary` tag
* Add a `@author` tag if you like
* add the `@api private` tag